code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -415,7 +415,7 @@ datum
 			result = "ldmatter"
 			required_reagents = list("plasma" = 1, "radium" = 1, "carbon" = 1, "stabiliser" = 1)
 			result_amount = 4
-			mix_phrase = "The begins to glow in a dark purple."
+			mix_phrase = "The begins to glow a dark purple."
```

Comment:
```suggestion
			mix_phrase = "The drink begins to glow a dark purple."
```

Reply:
This one is truly epic that I missed it. Wild. 👍